### PR TITLE
Fix karma debug

### PR DESCRIPTION
**Pull request by Ijmir**
added a fix for karma debugging. 
another fix will be pushed to generator-sublime to support invocation of karma with gulp.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -420,6 +420,11 @@ If you just want to run karma and are not interested yet in linting your files y
 gulp karma
 ```
 
+If you wish to debug the code please use the --debug flag
```
Comment (thaiat) [inline README.md]: put the full command here so it is easier for people to understand

File: README.md [hunk 2]
```diff
@@ -420,6 +420,11 @@ If you just want to run karma and are not interested yet in linting your files y
...
+```
```
Comment (thaiat) [inline README.md]: add bash after ```, it will color format the content

File: templates/app/karma.conf.js [hunk 3]
```diff
@@ -1,6 +1,28 @@
 'use strict';
 
+var args = require('yargs').argv;
+
 module.exports = function(config) {
+    var debug = false;
+    try {
+        debug = JSON.parse(args._[0]).debug;
+    } catch(ex) {}
```
Comment (thaiat) [inline templates/app/karma.conf.js]: ex is not used here so you can remove it
better to use `err` in node.js, ex is too C#

**Comment by thaiat** [comment]
I don't see a `Fixes...` in the comment of the karma commit

after you made the small changes in the comment code, make sure your PR has only 2 commits, one karma, the other one for the readme

**Comment by thaiat** [comment]
:shipit:

File: README.md (hunk 2 above)
Comment (Ijmir) [inline README.md]: we agreed that this is ok

**Comment by thaiat** [comment]
:shipit:

**Pull request merged**